### ISSUE TITLE
invalidate_hard_deletes=True directive does not work 

### ISSUE THREAD

**Issue by gbonazzoli** [oracle/dbt-oracle#37]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

If we have the following snapshot's confg: 

```
{{
    config(
      target_database='psbi',
      target_schema='dwh',
      unique_key='ID',

      strategy='check',
      check_cols='all',
      invalidate_hard_deletes=True
    )
}}
```

and run 

`dbt snapshot `

we obtain the following error:

```
04:46:44  Database Error in snapshot fct_piano_car (snapshots/fct_piano_car.sql)
04:46:44    ORA-00905: missing keyword
```

### Environment

```markdown
- OS: Ubuntu 20.04
- Python:  3.8.10
- dbt: 1.1.1
- dbt-oracle: 1.1.1
```

### What Oracle database version are you using dbt with?

19c

### Additional Context

none

**Comment by aosingh** [comment]
@gbonazzoli 

The fix is identifier and currently the PR is in-review. I will let you know once we release this. 

Thank you,
Abhishek Singh

**Comment by aosingh** [comment]
Hi @gbonazzoli 

This is fixed in dbt-oracle==1.1.2. We have also added a test case in our suite to test invalidation of hard deletes during dbt snapshots. 

You can install this version using 

```
pip install dbt-oracle==1.1.2
```
Please test this and let me know if you have any questions.

**Comment by aosingh** [comment]
Closing this issue. This was fixed in dbt-oracle==1.1.2. We have also added a test case in our suite to test invalidation of hard deletes during dbt snapshots.

**Comment by gbonazzoli** [comment]
@aosingh, well done ! 

I confirm you that every things is now OK! 

Tested and used in production with the following software versions:

```
root@new-airflow-dbt:~ # dbt --version
02:30:07  oracle adapter: Running in cx mode
Core:
  - installed: 1.3.0
  - latest:    1.3.0 - Up to date!

Plugins:
  - oracle:   1.3.0 - Up to date!
  - postgres: 1.3.0 - Up to date!
```
